<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="PKLAHL743" failures="1" tests="6" name="Test" time="78.634" errors="0" timestamp="2023-08-01T15:05:21 PKT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="17.118"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="15.937"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="25.023"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="16.8"/>
  <testcase classname="com.restassured.api.restAPI" name="GetUserDetails" time="1.883"/>
  <testcase classname="com.restassured.api.restAPI" name="verifyPetCreation" time="1.702">
    <failure type="java.lang.AssertionError" message="expected [jackie1] but found [jackie]">
      <![CDATA[java.lang.AssertionError: expected [jackie1] but found [jackie]
at com.restassured.api.restAPI.verifyPetCreation(restAPI.java:78)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- verifyPetCreation -->
</testsuite> <!-- Test -->
